client/src/components/StreamField/blocks/BaseSequenceBlock.js

```diff
@@ -143,7 +143,7 @@ export class BaseSequenceChild extends EventEmitter {
           ? `data-contentpath="${h(this.id)}"`
           : 'data-contentpath-disabled'
       }>
-        <input type="hidden"  name="${this.prefix}-deleted" value="">
+        <input type="hidden" name="${this.prefix}-deleted" value="">
```

Comment:
Try to avoid unrelated changes, this has actually been fixed up on main I think anyway.